src/async_hta_service.hpp

```diff
@@ -442,6 +450,89 @@ class AsyncHtaService
         return it.first->second;
     }
 
+private:
+    class Stats
```

Comment:
Move that class into its own header file. There's no need to eat the increase file here.